### ISSUE TITLE
Project description suggestion

### ISSUE THREAD

**Issue by mk3-14159** [karchuntan/istio-service-mesh#2]
# Comments
If the purpose of this micro-innovation is to provide a demo / boilerplate of how to resolve the complexities of juggling multiple microservices applications, communicating the project's business value will be crucial in the project description to gain more marketability & visibility.

# Proposed Project Description 
> A collection of implementations that demonstrates an Istio service mesh with FastAPI as one microservice. It uses Docker and Dockerhub for containerization, Kubernetes (K3s) for orchestration

# Proposed Topics 
1. python
2. fastapi
3. microservices
4. docker
5. dockerhub
6. kubernetes
7. k3s
8. istio
9. service-mesh
10. containerization
11. orchestration
12. istio-service-mesh
13. api-development
14. cloud-native
15. devops
16. kubernetes-deployement

**Comment by karchuntan** [comment]
Yeah, I agree, we can create a new branch to edit the documentation based on your suggestions and convert this issue to PR for tracking purpose.
